simple-weather/simple-weather/View/ForecastCell.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ForecastCell.swift
+//  simple-weather
+//
+//  Created by egmars.janis.timma on 25/04/2019.
+//  Copyright © 2019 egmars.janis.timma. All rights reserved.
+//
+
+import UIKit
+
+class ForecastCell: UITableViewCell {
+    
+    @IBOutlet var forecastPic: UIImageView!
+    @IBOutlet var forecastWeather: UILabel!
+    @IBOutlet var forecastMinTemp: UILabel!
+    @IBOutlet var forecastMaxTemp: UILabel!
+    @IBOutlet var forecastDay: UILabel!
+    
+
+    override func awakeFromNib() {
```

Comment:
In your case you do not need `override awakeFromNib()` or `override setSelected(_ :animated:)`, it's boilerplate added by creating `UITableViewCell` subclass